Cargo.toml

```diff
@@ -12,3 +12,4 @@ categories = ["no-std"]
 
 [dependencies]
 bitflags = "1.3.2"
+uuid = { version = "0.8.2", default-features = false }
```

Comment:
I don't really feel like pulling in this crate as "required" since in the whole crate its only used once and only for a `.into()` implementation. This can be made into a separate feature instead which if enabled will pull in the crate and implement the into conversion method. Or on the other hand, just make the user pull in the crate by themselves which is more neat IMO since you can construct the UUID crate struct with the stivale GUID easily outside of the crate. Then you can make the field `part_guid` private and make a method instead which returns a reference to the stivale GUID to ensure no one mutates the fields in the structure.

Reply:
Alright; I'll make `uuid` optional. I still would like to have it, since it feels more convenient to be able to call `.into()` as opposed to writing out the whole constructor, but I'll bring it in as optional so people who don't need it don't get it.

Reply:
One more thing: it doesn't seem typical to make variables private, judging by the rest of the file; do you have a specific reason for this case?